### PR TITLE
prevent user watch events from updating channel state

### PR DESCRIPTION
**Pull request by DanC5**
# Submit a pull request

### 🎯 Goal

Prevent the message list from re-rendering when users join/leave a channel

### 🛠 Implementation details

Do not initiate an action to update the state returned from the channel reducer when user watch events are received

### 🎨 UI Changes

None

### CODE REVIEW

**Review by zacheryconverse — APPROVED** [review]

File: examples/typescript/yarn.lock [hunk 1]
```diff
@@ -10082,6 +10073,14 @@ react-virtuoso@^1.5.8:
     react-app-polyfill "^1.0.6"
     resize-observer-polyfill "^1.5.1"
 
+react-virtuoso@^2.2.0:
```
Comment (petyosi) [inline examples/typescript/yarn.lock:10076]: having two react-virtuosos in yarn.lock does not seem right

Comment (DanC5) [reply examples/typescript/yarn.lock:10076]: oh interesting, that second one might be coming from the MML library, let me check...

Comment (DanC5) [reply examples/typescript/yarn.lock:10076]: yep it's coming from `mml-react` dependency

Comment (DanC5) [reply examples/typescript/yarn.lock:10076]: should we make it a peer dep in that repo?

Comment (petyosi) [reply examples/typescript/yarn.lock:10076]: I would rather upgrade it TBH

Comment (DanC5) [reply examples/typescript/yarn.lock:10076]: ok i'll upgrade it in `mml-react`

**Pull request merged**